bitcoin/src/blockdata/block.rs

```diff
@@ -41,7 +41,15 @@ pub type BlockInterval = BlockHeightInterval;
 
 internal_macros::impl_hashencode!(BlockHash);
 
-internal_macros::impl_consensus_encoding!(Header, version, prev_blockhash, merkle_root, time, bits, nonce);
+internal_macros::impl_consensus_encoding!(
+    Header,
+    version,
+    prev_blockhash,
+    merkle_root,
+    time,
+    bits,
+    nonce
+);
```

Comment:
Shall we fmt skip this one? I can do it as a follow up if you think it worth it.